core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -705,7 +713,7 @@ public Builder numberOfReplicas(int numberOfReplicas) {
         public int numberOfReplicas() {
             return settings.getAsInt(SETTING_NUMBER_OF_REPLICAS, -1);
         }
-        
+
```

Comment:
maybe we could revert these changes given that it's only formatting and make them separately to keep the PR smaller?